mimic/core.py

```diff
@@ -35,13 +35,16 @@ def __init__(self, clock, apis, domains=()):
             and determining timestamps.
         :type clock: :obj:`twisted.internet.interfaces.IReactorTime`
 
-        :param apis: an iterable of all :obj:`IAPIMock`s that this MimicCore
-            will expose.
+        :param apis: an iterable of all :obj:`IAPIMock` and
+            :obj:`IAPIExternalAPI` mocks that this MimicCore will expose.
 
         :param domains: an iterable of all :obj:`IAPIDomainMock`s that this
             MimicCore will expose.
         """
-        self._uuid_to_api = {}
+        self._uuid_to_api = {
```

Comment:
This is also a style thing, but I'd prefer you block on it: I *really* don't like the practice of pretending that raw dictionaries are structured objects.  It's hard to document; a simple `:ivar:` ReST or epytext field can document what `_uuid_to_api` is, but there isn't even any syntax for explaining what `_uuid_to_api["internal"]` is.  It is harder on IDEs, because they can't "see" the structure of this attribute.  It's harder on grep, because you could use single or double quotes, or a constant.  It's harder on linters because they can't verify that attribute names match; it's a dict, so any key of the right type should be valid!  It's harder to catch versioning issues, because we're preserving the name `self._uuid_to_api` but giving its keys and values totally different types.

I could go on :).

For these reasons, and more, please replace `self._uuid_to_api['external']` with `self._uuid_to_external` and add `:ivar:`s for each of these, then go ahead and land on master; this is my last bit of feedback.

Thanks for participating in Mimic's development 😄!